pkg/client/cache/ssdb/ssdb.go

```diff
@@ -23,66 +25,61 @@ func NewSsdbCache() cache.Cache {
 	return &Cache{}
 }
 
-// Get gets a key's value from memcache.
-func (rc *Cache) Get(key string) interface{} {
+// Get gets a key's value from ssdb.
+func (rc *Cache) Get(key string) (interface{}, error) {
+	return rc.GetWithCtx(context.Background(), key)
+}
+
+// GetWithCtx gets a key's value from ssdb.
+func (rc *Cache) GetWithCtx(ctx context.Context, key string) (interface{}, error) {
+	rc.conn.Do()
 	if rc.conn == nil {
 		if err := rc.connectInit(); err != nil {
-			return nil
+			return nil, nil
 		}
 	}
-	value, err := rc.conn.Get(key)
-	if err == nil {
-		return value
-	}
-	return nil
+	return rc.conn.Get(key)
 }
 
-// GetMulti gets one or keys values from memcache.
-func (rc *Cache) GetMulti(keys []string) []interface{} {
+// GetMulti gets one or keys values from ssdb.
+func (rc *Cache) GetMulti(keys []string) ([]interface{}, error) {
+	return rc.GetMultiWithCtx(context.Background(), keys)
+}
+
+// GetMultiWithCtx gets one or keys values from ssdb.
+func (rc *Cache) GetMultiWithCtx(ctx context.Context, keys []string) ([]interface{}, error) {
 	size := len(keys)
 	var values []interface{}
 	if rc.conn == nil {
 		if err := rc.connectInit(); err != nil {
-			for i := 0; i < size; i++ {
-				values = append(values, err)
-			}
-			return values
+			return values, err
 		}
 	}
 	res, err := rc.conn.Do("multi_get", keys)
 	resSize := len(res)
 	if err == nil {
-		for i := 1; i < resSize; i += 2 {
-			values = append(values, res[i+1])
-		}
-		return values
+		return values, err
 	}
-	for i := 0; i < size; i++ {
-		values = append(values, err)
+	for i := 1; i < resSize; i += 2 {
+		values = append(values, res[i+1])
 	}
```

Comment:
I can not understand those codes. What's the purpose and I can not find related unit tests